### ISSUE TITLE
Employee List View Screen Development

### ISSUE THREAD

**Issue by procupid** [kanwalnainsingh/OpenDesk#108]
The Design is ready and available in both mobile and desktop versions so that screens are responsive.

### **[FIGMA FILE](https://www.figma.com/file/NihC7mPNWDLTTfOyKpChhN/BookOpendesk?node-id=7%3A22567)**

DESKTOP VERSION
![Landing Page](https://user-images.githubusercontent.com/67896359/100637419-dc3c2580-3358-11eb-94a9-ef39c3e852a8.png)

MOBILE VERSION
![Employee page](https://user-images.githubusercontent.com/67896359/100637428-de9e7f80-3358-11eb-884c-128b5ccccdb7.png)

**Comment by CarlosFabianG** [comment]
Hello, can I work on this one?

**Comment by kanwalnainsingh** [comment]
> Hello, can I work on this one? 

@CarlosFabianG yes you can pick this, let's us know if you need any help. Join slack for more details https://www.bookopendesk.com/
